doc/make.py

```diff
@@ -204,17 +204,11 @@ def _add_redirects(self):
 
                 path = os.path.join(BUILD_PATH, "html", *row[0].split("/")) + ".html"
 
-                try:
-                    title = self._get_page_title(row[1])
-                except Exception:
-                    # the file can be an ipynb and not an rst, or docutils
-                    # may not be able to read the rst because it has some
-                    # sphinx specific stuff
-                    title = "this page"
+                title = self._get_page_title(row[1])
```

Comment:
Is there a reason you are changing this? Is the comment explaining it no longer true?

Reply:
looking to see in the CI if something more specific is raised